<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="scoreBoard">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>55</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="player1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="player1Score">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>10</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="separator_2">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="player2Score">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="player2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="TicTacToe" name="gameBoard" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>15</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Game"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_Game"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="actionNew_Game">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/newGame.png</normaloff>:/newGame.png</iconset>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
   <property name="toolTip">
    <string>Start a new game</string>
   </property>
   <property name="shortcut">
    <string>Meta+N</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/exit.png</normaloff>:/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit the program</string>
   </property>
   <property name="shortcut">
    <string>Meta+Q</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TicTacToe</class>
   <extends>QWidget</extends>
   <header>tictactoe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
